WordPress/Sniffs/Theme/NoHardCodedUrlsSniff.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * WordPress_Sniffs_Theme_NoHardCodedUrlsSniff File Doc Comment
+ *
+ * @category WordPress_Sniffs_Theme_NoHardCodedUrlsSniff
+ * @package  PHP_Codesniffer
+ * @author   khacoder
+ * @license  http://www.gnu.org/copyleft/gpl.html GNU General Public License
+ * @link     http://kevinsspace.ca
+ */
+
+/**
+ * WordPress_Sniffs_Theme_NoHardCodedUrlsSniff Class Doc Comment
+ *
+ * WARNING Verify that no hard-coded URLs are found. Manual verification necessary.
+ * An exception is made for the Author URI and (Theme) URI as set in the style.css header
+ * as well as links to wordpress.org. Links in the text of PHP/JS comments should be
+ * excluded from this check.
+ *
+ * @category WordPress_Sniffs_Theme_NoHardCodedUrlsSniff
+ * @package  PHP_Codesniffer
+ * @author   khacoder
+ * @license  http://www.gnu.org/copyleft/gpl.html GNU General Public License
+ * @link     http://kevinsspace.ca
+ */
```

Comment:
> What do you think having the `@category` as WordPress_Theme?

No - from all appearances `@category` is very high level (higher level than `@package`) and should be `PHP` as it's a PHP project.